third_party/3/jwt/contrib/algorithms/__init__.pyi

```diff
@@ -0,0 +1,8 @@
+from typing import Callable, Any, Union
+from hashlib import _DataType
+
+# In reality, _HashAlg is a function of the type
+# that hashlib.sha256/384/512 are, but there doesn't
+# seem to be a consistent exportable name that we can reference
+# for that, so we'll just say it's a Callable here.
```

Comment:
These stubs are in stdlib/3 so they should only use the Python 3 version. When you switch to what I suggested does it still fail?